nimble/host/mesh/src/model_cli.c

```diff
@@ -181,6 +177,7 @@ int bt_mesh_gen_onoff_set(u16_t net_idx, u16_t addr, u16_t app_idx,
 
 	err = cli_wait(&gen_onoff_cli, &param, OP_GEN_ONOFF_STATUS);
 done:
+	transaction_id++;
```

Comment:
do I understand it correctly that err will be non zero when timeout on ACK will happen? If so I'm not sure if we should increase transaction_id

Reply:
Correct, fixed.